unit_tests/pytest.ini

```
@@ -0,0 +1,10 @@
+# Must reside in root of directory of in folder where test are executed from.
+[pytest]
+markers =
+    ; <marker name>: <optional marker description>
+    ; <marker 2 name>:
+    ; .
+    ; .
+    ; .
+    ; <etc>:
+    op_lib_monipulation:
```

there's no hidden meaning, just a typo, right? 
```suggestion
    op_lib_manipulation:
```